lib/Controller/WizardController.php

```diff
@@ -63,7 +64,8 @@ public function disable() {
 	 * @return TemplateResponse
 	 */
 	public function show() {
-		$theming = \OC::$server->getThemingDefaults();
+		/** @var Defaults $theming */
+		$theming = \OC::$server->query(Defaults::class);
```

Comment:
I don't really get why we need to change this. I would like to avoid this. Because it basically means the API changed....